functions/slaViolationChecker/src/app/lib/eventMapper.js

```diff
@@ -46,10 +90,60 @@ exports.checkRemovedByTTL = (kinesisEvent) => {
   return false;
 };
 
-exports.mapEvents = (events) => {
+exports.mapEvents = async (events) => {
+  let ops = [];
+  for (let i = 0; i < events.length; i++) {
+    const dynamoDbOps = await mapPayload(events[i]); // we are adding an array, not a single element
+    ops = ops.concat(dynamoDbOps);
+  }
+  return ops;
+};
+
+// SQS flow
+const makeUpdateOpFromSQS = (event, endTimestamp) => {
```

Comment:
@FeliceMente potrebbe essere unificata a `makeUpdateOp` con un parametro in più che definisce se viene da SQS o da Kinesis

Reply:
giusto. Fatto!